src/program/services/scrapers/shared.py

```
@@ -2,269 +2,114 @@
+    
+    return settings
 
 
 def parse_results(
     item: MediaItem,
     results: dict[str, str],
-    ranking_overrides: RankingOverrides | None = None,
-    manual: bool = False,
+    relaxed_validation: bool = False,
+    overrides: dict | None = None,
+    max_bitrate_override: int | None = None,
```

Ruff (0.14.10)

Unused function argument: `max_bitrate_override`

(ARG001)